### DISCUSSION TITLE
Access to Android Library Source Files?

### DISCUSSION THREAD

**Discussion by ali-soulside** [yasirkula/UnityNativeCamera#54] [General]
Hi, thank you for making this awesome plugin available to us.

I wanted to know if I can get access to the plugin's android source files? The reason I need access is because I want to see if it is possible to make the video editable(trimming only) after it's recorded. I was able to make this editing work on iOS, but not yet on Android.

**Comment by yasirkula** [comment]
Of course: https://github.com/yasirkula/UnityNativeCamera/tree/master/.github/AAR%20Source%20(Android)

**Reply by ali-soulside** [reply]
Thank you!
